executor/runtime/types/container.go

```diff
@@ -434,6 +437,15 @@ func (c *TitusInfoContainer) updateLogAttributes(titusInfo *titus.ContainerInfo)
 		c.logUploadRegexp = uploadRegexp
 	}
 
+	finalUploadMatchRegexStr, ok := titusInfo.GetPassthroughAttributes()[finalUploadMatchRegex]
```

Comment:
Might want to use `getPassthroughBool()` here, to be consistent with other passthrough bools.